addons/l10n_es_edi_facturae/tests/test_edi_xml.py

```diff
@@ -274,10 +274,7 @@ def test_import_multiple_invoices(self):
 
         moves += self.env['account.move'].search([('ref', '=', 'INV/2023/00006'), ('company_id', '=', self.company_data['company'].id)], limit=1)
 
-        partner = self.env['res.partner'].search([
-            ('name', '=', 'Azure Interior'),
-            ('email', '=', 'azure.Interior24@example.com'),
-        ])
+        partner = self.env.ref('base.res_partner_12')
```

Comment:
We should create a new partner in `setUpClass`, as well as the xml files to match that new partner (I see the same issue with `import_withholding_invoices.xml` even though we don't check the partner)